agnpy/tests/test_spectra.py

```diff
@@ -538,10 +538,127 @@ def test_init(self, p, gamma_c, gamma_min, gamma_max):
         assert u.isclose(n_gamma_1, epwl_e(1), atol=0 * u.Unit("cm-3"), rtol=1e-2)
         assert u.isclose(n_gamma_1, epwl_p(1), atol=0 * u.Unit("cm-3"), rtol=1e-2)
 
+class TestExpCutoffBrokenPowerLaw:
+    """Class grouping all tests related to the ExpCutoffBrokenPowerLaw spectrum."""
+
+    def test_call(self):
+        """Assert that outside the bounding box the function returns 0."""
+        gamma = np.logspace(0, 8)
+        ebpwl = ExpCutoffBrokenPowerLaw()
+        values = ebpwl(gamma).value
+        condition = (ebpwl.gamma_min <= gamma) * (gamma <= ebpwl.gamma_max)
+        # check that outside the boundaries values are all 0
+        assert not np.all(values[~condition])
+
+    @pytest.mark.parametrize("p1", np.arange(1, 2, 0.5))
+    @pytest.mark.parametrize("p2", np.arange(3, 4, 0.5))
+    @pytest.mark.parametrize("gamma_b", np.logspace(3, 5, 3))
+    @pytest.mark.parametrize("gamma_min", np.logspace(0, 2, 3))
+    @pytest.mark.parametrize("gamma_max", np.logspace(6, 8, 3))
+    @pytest.mark.parametrize("gamma_c", np.logspace(3, 5, 3))
```

Comment:
I am just wondering what is the effect of having `gamma_c` < `gamma_b`, that the spectrum looks like a simple cutoff powerlaw?

Reply:
Yes, from the point where gamma_c = gamma_b, they look alike but they're not exactly the same (even if you put the normalization constant of the exp cut off equal to the one of exp bkn cut off times the gamma_b to the power of p) . They are, thought, for very high values of gamma